man/kernel-command-line.xml

```diff
@@ -322,6 +322,14 @@
       </varlistentry>
 
       <varlistentry>
+              <term><varname>systemd.default_timeout_start_sec=</varname></term>
+
+              <listitem>
+                      <para>Overwrites the default start job timeout (DefaultTimeoutStartSec in system.conf) at boot.</para>
```

Comment:
Please enclose the `DefaultTImeoutStartSec` in `<varname></varname>`. Also, please replace `system.conf` by a proper man page reference to `systemd-system.conf(5)`. i.e. with `<citerefentry>` and stuff.